gradle/dependencies.gradle

```diff
@@ -71,7 +71,7 @@ else
 // Java versions.
 // We always use the `inline` strategy.
 String mockitoArtifactName
-if (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_11))
+if (mockitoVersion == "5.10.0")
```

Comment:
Could you please use "startWith" to make sure we have correct mockito artifacts?

Reply:
yeah sure, I also rearranged the order to make sure  if mockito 6 still use the same structure of version 5. we don't need to change these piece of code.